src/main/resources/dto/item.json

```diff
@@ -0,0 +1,435 @@
+    },
+    "hrid": {
+      "type": "string",
+      "description": "The human readable ID, also called eye readable ID. A system-assigned sequential alternate ID"
+    },
+    "holdingsRecordId": {
+      "type": "string",
+      "description": "ID of the holdings record the item is a member of."
+    },
+    "instanceId": {
+      "type": "string",
+      "description": "ID of the instance."
+    },
+    "formerIds": {
+      "type": "array",
+      "description": "Previous identifiers assigned to the item",
+      "items": {
+        "type": "string"
+      },
+      "uniqueItems": true
+    },
+    "discoverySuppress": {
+      "type": "boolean",
+      "description": "Records the fact that the record should not be displayed in a discovery system"
+    },
+    "accessionNumber": {
+      "type": "string",
+      "description": "Also called inventar number"
+    },
+    "barcode": {
+      "type": "string",
+      "description": "Unique inventory control number for physical resources, used largely for circulation purposes"
+    },
+    "itemLevelCallNumber": {
+      "type": "string",
+      "description": "Call Number is an identifier assigned to an item, usually printed on a label attached to the item. The call number is used to determine the items physical position in a shelving sequence, e.g. K1 .M44. The Item level call number, is the call number on item level."
+    },
+    "itemLevelCallNumberPrefix": {
+      "type": "string",
+      "description": "Prefix of the call number on the item level."
+    },
+    "itemLevelCallNumberSuffix": {
+      "type": "string",
+      "description": "Suffix of the call number on the item level."
+    },
+    "itemLevelCallNumberTypeId": {
+      "type": "string",
+      "description": "Identifies the source of the call number, e.g., LCC, Dewey, NLM, etc."
+    },
+    "effectiveCallNumberComponents": {
+      "type": "object",
+      "description": "Elements of a full call number generated from the item or holding",
+      "properties": {
+        "callNumber": {
+          "type": "string",
+          "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item.",
+          "readonly": true
+        },
+        "prefix": {
+          "type": "string",
+          "description": "Effective Call Number Prefix is the prefix of the identifier assigned to an item or its holding and associated with the item.",
+          "readonly": true
+        },
+        "suffix": {
+          "type": "string",
+          "description": "Effective Call Number Suffix is the suffix of the identifier assigned to an item or its holding and associated with the item.",
+          "readonly": true
+        },
+        "typeId": {
+          "type": "string",
+          "description": "Effective Call Number Type Id is the call number type id of the item, if available, otherwise that of the holding.",
+          "$ref": "uuid.json",
```

Comment:
There is an existing file `src/main/resources/swagger.api/schemas/uuid.json`. Is it possible to use it here?